lib/semanticchess.test.js

```diff
@@ -178,60 +177,98 @@ describe('Semantic Chess', function() {
   
   describe('promotion', function() {
 	  
+	const params = {
+		url: 'http://example.org/myGame',
+		moveBaseUrl: 'http://example.org/myfile',
+		userWebId: 'http://example.org/#me',
+		opponentWebId: 'http://example.org/#other',
+		name: 'My game',
+		uniqid: Utils.customUniqid()
+	}
+	  
     it('valid moves', async function() {
-		var semanticChess = new SemanticChess({
-		  url: 'http://example.org/myGame',
-		  moveBaseUrl: 'http://example.org/myfile',
-		  userWebId: 'http://example.org/#me',
-		  opponentWebId: 'http://example.org/#other',
-		  name: 'My game',
-		  uniqid: Utils.customUniqid()
-		});
 		
 		//POSITIVE TESTS
+		
 		//Test in the middle of the board (white pawn)
-		semanticChess.chess = new chess.Chess('8/4P3/8/8/8/8/K6k/8 w - - 0 1');
+		var semanticChess = new SemanticChess({
+		  params,
+		  startPosition: '8/4P3/8/8/8/8/K6k/8 w - - 0 1'
+		});
```

Comment:
The constructor is wrong here. To achieve what you want to do you should do `params.startPosition= "..."` and then do `new SemanticChess(params)`.